docs/configuration/single-process-config-blocks-gossip-1.yaml

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
We're used to add an integration test for each getting started config we do provide. May you add a test with this config to `integration/getting_started_single_process_config_test.go`, please?

Reply:
Added (although in a different filename).